components/footer/favs/favsBtn.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from "./favsBtn.module.css";
+/* import FavsImg from "./favsBtn.stories"; */
+
+function FavsBtn() {
+  return (
+    <a href="http://localhost:3000/">
```

Comment:
```suggestion
    <a href="#">
```
Remove the link to the localhost as a placeholder